templates/music/album_detail.html

```diff
@@ -12,58 +12,131 @@
-        <div class="track-listing">
-            {% for disc in album.discs %}
-                <ol class="ol-center">
-                    {% for track in disc %}
-                        <li>{{ track.name }}{% if track.duration %} [{{ track.duration|trackdurationformat }}]{% endif %}</li>
-                    {% endfor %}
-                </ol>
-            {% endfor %}
-        </div>
-    {% endif %}
     {% if album.albumbio %}
-        <h4>About</h4>
-        <div class="break-small">
-            <hr>
-        </div>
         <div class="album-bio">
             <p>{{ album.albumbio.bio|markdown }}</p>
         </div>
-    {% endif %}
-
-    {% if album.marketplaceurl_set.all %}
-        <h4>LISTEN / PURCHASE</h4>
         <div class="break-small">
             <hr>
         </div>
-        <div class="marketplace-container">
-            {% for marketplace_url in album.marketplaceurl_set.all %}
-                <a href="{{ marketplace_url.affiliate_url }}" target="_blank">
-                    {% with "img/marketplace/"|add:marketplace_url.medium|add:".svg" as medium_image %}
-                        <img class="marketplace-icons" src="{% static medium_image %}" alt="{{ marketplace_url.get_medium_display }}" />
-                    {% endwith %}
-                </a>
-            {% endfor %}
-        </div>
-        <br />
     {% endif %}
 
+    <h4>Listen</h4>
+    <div class="marketplace-container">
+        {% for marketplace_url in album.marketplaceurl_set.all %}
+            <a href="{{ marketplace_url.url }}" target="_blank">
+                {% with "img/marketplace/"|add:marketplace_url.medium|add:".svg" as medium_image %}
+                    <img class="marketplace-icons" src="{% static medium_image %}" alt="{{ marketplace_url.get_medium_display }}" />
+                {% endwith %}
+            </a>
+        {% endfor %}
+    </div>
+    {% if album.release_date %}
+        <h5 class="padding">Release date: {{ album.release_date }}</h5>
+    {% endif %}
+    <div class="break-small">
+        <hr>
+    </div>
+    <div>
     {% if user_is_investor %}
-        <h4>FOR INVESTORS</h4>
-        <div class="break-small">
-            <hr>
-        </div>
         <div class="download-container">
             <a class="download-button" href="{{ album.audio.get_temporary_url }}" download>
                 Download
                 <i class="fa fa-download" aria-hidden="true"></i>
             </a>
         </div>
+        <div class="break-small">
+            <hr>
+        </div>
     {% endif %}
+
+    <h3>REVENUE</h3>
+        <div class="bar-chart">
+            <canvas id="line-canvas" height="650px" width="1000px"></canvas>
+        </div>
+    </div>
+
+    <br/>
+    <br/>
+    <h3>Streams / Downloads</h3>
+    {% for disc in album.discs %}
+    {% for track in disc %}
+        <div class="bar-chart">
+                <canvas id="bar-canvas" height="450" width="600"></canvas>
+        </div>
+        {% block extrajs %}
```

Comment:
The entire `{% block extrajs %}` and all JS should come after the current content block that we're in.